Source/WebKit/NetworkProcess/cache/NetworkCacheStorage.cpp

```diff
@@ -138,9 +138,9 @@ bool Storage::ReadOperation::finish()
 struct Storage::WriteOperation {
     WTF_MAKE_TZONE_ALLOCATED(Storage::WriteOperation);
 public:
-    WriteOperation(Storage& storage, const Record& record, MappedBodyHandler&& mappedBodyHandler, CompletionHandler<void(int)>&& completionHandler)
+    WriteOperation(Storage& storage, const Record& inputRecord, MappedBodyHandler&& mappedBodyHandler, CompletionHandler<void(int)>&& completionHandler)
         : storage(storage)
-        , record(record)
+        , record({ crossThreadCopy(inputRecord.key), inputRecord.timeStamp, inputRecord.header, inputRecord.body, inputRecord.bodyHash })
```

Comment:
Should we just define `isolatedCopy()` on record so we can just do `record(crossThreadCopy(record))`?

Reply:
Yes, we could do that too; will update